src/WinGetUtil/Exports.cpp

```diff
@@ -356,7 +357,9 @@ extern "C"
             }
             catch (const ManifestException& e)
             {
-                WI_SetFlagIf(validationResult, WinGetValidateManifestResult::DependenciesValidationFailure, !e.IsWarningOnly());
+                validationResult |= WinGetValidateManifestResult::DependenciesValidationFailure;

```

Comment:
We are converting these from warnings to actual errors. Is there anybody that could break due to this change?

Reply:
Even if we don't have warning level dependencies now, we should still only set the error when ther level is error, in case we have warning level in the future